package/src/components/Chat/Chat.tsx

```diff
@@ -226,8 +217,6 @@ const ChatWithContext = <
     enableOfflineSupport,
   });
 
-  if (loadingTranslators) return null;
```

Comment:
Make sure to test it by adding some delays to `streami18n.getTranslators()` method. Just to make sure whats the behavior when `translators` is not ready